tests/core/interfaces-gpio-chardev/task.yaml

```diff
@@ -0,0 +1,136 @@
+summary: Ensure that the GPIO chardev interface works.
+
+details: |
+    The gpio-chardev interface allows read/write access to specified GPIO chip lines.
+
+systems: [ubuntu-core-2*]
+
+prepare: |
+    # emulate GPIO chips using gpio-sim module
+    cat << \EOF > gpio-sim.service
```

Comment:
What's the idea with the backslash?

Reply:
My bad, This was copy-pasted from another test. This is equivalent to quoting, now that it is removed I don't need the sed replacement below.